modules/Attendance/src/AttendanceView.php

```diff
@@ -78,6 +78,7 @@ public function __construct(Core $gibbon, Connection $pdo, SettingGateway $setti
         $this->settingGateway = $settingGateway;
 
         // Collect the current IDs of the user
+        global $session;
```

Comment:
For readability sake, the defacto standard is to place the "global" statement at the top of a function.